programs/server/Server.cpp

```diff
@@ -646,6 +650,44 @@ static void sanityChecks(Server & server)
     }
 }
 
+#if defined(OS_LINUX)
+static void _sd_notify(const char*command)
```

Comment:
I'm not a C++ developer, so made in C style. It works, but looks like something outdated.